editor/test/integration/scope_test.clj

```diff
@@ -45,7 +45,7 @@
           old-count (node-count (g/graph graph-id))
           old-node-ids (set (ig/node-ids (g/graph graph-id)))
           old-basis (g/now)
-          mem-resource (project/make-embedded-resource project resource-type-name inline-resource)]
+          mem-resource (project/make-embedded-resource project :editable resource-type-name inline-resource)]
```

Comment:
The `make-embedded-resource` function now takes an additional argument that specifies whether the embedded resource should be `:editable` or `:non-editable`.